biblio.Rmd

```diff
@@ -115,3 +115,39 @@ pandoc homework.md --bibliography=mybib.bib -o .\Outputs\output.html
 
 where `--bibliography=mybib.bib` specifies that you want to use your `mybib.bib`, expected at the project root.
 This would be the file that you'd have exported from Zotero.
+
+## Resources on How to Start Contributing to Open Source
+
+Mastering the entry into open source can be frightening. To make your contribution easier, a collection of links has been compiled here. First of all, it makes sense to spend a bit of time on blogs on this topic.
+
+### Blogs and Anchors
+
+A **First Timers Only: For absolute Newbies**
```

Comment:
you can use `#### Fourth Level Headings` for this.

Reply:
**done**